src/renderer/widgets/RemoveProxy/ui/Confirmation.tsx

```diff
@@ -64,7 +65,9 @@ export const Confirmation = ({
         <hr className="border-filter-border w-full pr-2" />
 
         <DetailRow label={t('proxy.details.revokeAccessType')} className="pr-2">
-          <FootnoteText>{t(ProxyTypeName[proxyAccount.proxyType])}</FootnoteText>
+          <FootnoteText>
+            {t(ProxyTypeName[proxyAccount.proxyType]) || splitCamelCaseString(proxyAccount.proxyType)}
```

Comment:
I still don't understand why not to use `t(proxyUtils.getProxyTypeName(proxyType))`?
For not existing i18n token, `t()` will return original argument, which is the result of `proxyUtils.getProxyTypeName`

```javascript
function getProxyTypeName(proxyType: ProxyType | string): string {
  return ProxyTypeName[proxyType as ProxyType] || splitCamelCaseString(proxyType as string);
}
```